docs/docs-beta/docs/guides/tbd/running-local-ui-development.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Running Dagster locally
+description: How to run Dagster on your local machine.
+---
+
+# Running Dagster locally
+
+In this guide, we'll walk you through how to run Dagster on your local machine using the `dagster dev` command.  The `dagster dev` command launches the Dagster UI and the Dagster daemon, allowing you to start a full deployment of Dagster from the command line.
+
+## Locating your code
+
+Before starting local development, you need to tell Dagster how to find the Python code containing your assets and jobs.
+
+<Tabs>
```

Comment:
Worried that maybe we aren't giving people a strong opinion on what to do. What about this for now:

```suggestion
For a refresher on how to setup a Dagste project, follow our [Recommended Dagster Project Structure](/todo) guide.

<Tabs>
```